docs/DEPLOYMENT.md

```diff
@@ -171,21 +206,31 @@ metadata:
   name: ui
   namespace: tekton-hub
 data:
-  API_URL: API URL   <<< update this by api routes
+  API_URL: API URL   <<< update this by api route
   GH_CLIENT_ID: GH OAuth Client ID   <<< update this
 ```
 
-### Apply the manifests
+#### Apply the manifests
 
 ```bash
-kubectl apply -f config
+kubectl apply -f config/
 ```
 
-### Ensure pods are up and running
+This will create the deployment and service for UI.
+
+#### Ensure pods are up and running
 
 ```bash
 kubectl get pods
```

Comment:
same as above, can we also add namespace flag here as well and in all the commands?
or also we can change the context to the desired namespace, so that we don't have to pass namespace in every command.